src/main/kotlin/CheckString.kt

```diff
@@ -0,0 +1,20 @@
+object CheckString {
+    fun checkBlank(input: String) {
+        if (input.equals(" ") || input.isNullOrEmpty()) {
+            throw IllegalArgumentException("공백은 입력할 수 없습니다.")
+        }
+    }
+
+    fun checkOperator(splitString: List<String>): Boolean {
+        val operator = listOf("+", "-", "*", "/")
+        val eachSplitString = splitString.indices
+        for (i in eachSplitString) {
+            if (i % 2 == 1) {
+                if (!(splitString[i] in operator)) {
+                    throw IllegalArgumentException("연산자가 잘못되었습니다.")
+                }
+            }
+        }
```

Comment:
```suggestion
        for (i in 1 until eachSplitString step 2) {
                if (!(splitString[i] in operator)) {
                    throw IllegalArgumentException("연산자가 잘못되었습니다.")
                }
        }
```
step 이용하면 해당 숫자만큼 점프하면서 반복가능합니돠

Reply:
좋은 꿀팁 정말 감사합니다ㅎㅎ